frontend/packages/process-editor/src/components/Canvas/Canvas.tsx

```diff
@@ -25,26 +22,15 @@ export type CanvasProps = {
  */
 export const Canvas = ({ onSave }: CanvasProps): JSX.Element => {
   const { getUpdatedXml, isEditAllowed } = useBpmnContext();
-  const [isEditorView, setIsEditorView] = useState(false);
-
-  const toggleViewModus = (): void => {
-    setIsEditorView((prevIsEditorView) => !prevIsEditorView);
-  };
 
   const handleOnSave = async (): Promise<void> => {
     onSave(await getUpdatedXml());
   };
 
   return (
     <div className={classes.container}>
-      {isEditAllowed && (
-        <CanvasActionMenu
-          onSave={handleOnSave}
-          toggleViewModus={toggleViewModus}
-          isEditorView={isEditorView}
-        />
-      )}
-      <div className={classes.wrapper}>{isEditorView ? <BPMNEditor /> : <BPMNViewer />}</div>
+      {isEditAllowed && <CanvasActionMenu onSave={handleOnSave} />}
+      <div className={classes.wrapper}>{<BPMNEditor />}</div>
```

Comment:
We might removed a little too much here. We should keep the `<BPMNViewer>` but remove the possibility of toggling between them. We should use the `appLibCheck` to decide whether we should display the `<BPMNViewer>` or the `<BPMNEditor>`.

If the users are using app-lib less than 8, we should render the `<BPMNViewer>`, if they use version 8 or higher the `<BPMNEditor>` should be the one we are using. The reason for this is that Studio do not support editing on the old BPMN templates/files which is used in app-lib less than v8.

Reply:
In the PR I have been working on for ProcessEditor, I changed ```Canvas ``` to this: 
```tsx
<div className={classes.container}>
      <CanvasActionMenu onSave={handleOnSave} />
      <div className={classes.wrapper}>{isEditAllowed ? <BPMNEditor /> : <BPMNViewer />}</div>
  </div>
```